### PR TITLE
Conference Page

### PR DESCRIPTION
**Pull request by Ayokunnumi1**
These are the functionalities implemented in this project;
* Home Page and About Page
* Responsive design for mobile and desktop
* Interactive links in mobile menu and desktop navbar.
* Dynamic Creation of the Featured Speakers Section,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,149 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project. show your support by following my github account.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I give thanks to the Microverse team for creating this project.
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Your "Acknowledgements" section should gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,149 @@
...
+- **Home Page: An organized webpage containing major information about the event.**
+- **About Page:A linked webpage to Home Page giving more information about the event**
+- **Functionality: The mobile navbar, featured Speaker section were made easy for users to navigation the webpage with Javascript**
+- **Social Contact: A form allowing visitors to get in touch with me.**
+- **Responsive Design: The website is optimized for different devices and screen sizes.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link]()
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Your live demo currently doesn't show the live working version of your project hosted on github pages. Please i kindly suggest your host your project using github pages. The live version is important as it demo's your project for anyone checking it out :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Global Summit</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="mobile-navbar">
```
Comment (Efiamotu-1) [inline index.html:15]: - [ ] As shown in the images below on mobile view you are to only show the hamburger menu icon, but from your design your logo is also displaying. please kindly follow the provided project design 👍 
- project design
cc
- your design
cc ham

File: index.html [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+        </div>
+
+        <div class="desktop-header-section">
+            <div class="mobile-header-section">
+                <p class="header-theme">"You Never Fail Until You Stop Trying"</p>
+                <h1>Content Creator <br> Global Summit 2023</h1>
+                <p class="header-hero-desc">A joyful celebration believing in the value of openness and
+                    sharing, creating a positive change with people from over 80
+                    countries is taking place in October, in Korea.
+                </p>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] As shown in the project design below the text are much larger than what you currently have on your design, i will kindly suggest that your increase the text size to match closely the provided project design 👍 
- project design
ht
- your design
hts

File: index.html [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+                <span class="line"></span>
+            </div>
+            <div class="main-program-content">
+                <div class="main-program-short"><img src="images/lecture-img.png" alt="">
+                    <h3>Lecture</h3>
+                    <p class="main-program-desc">Listen to the speakers from various countries about the messages of
+                        sharing and opening</p>
+                </div>
+                <div class="main-program-short"><img src="images/exhibition-img.png" alt="">
+                    <h3>CC Exhibition</h3>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] Please kindly note that for each program the white border only becomes visible when a user hovers around it as shown in the provided design template below. However what you have is the white borders displaying and no spaces between each program. Please kindly make the necessary adjustments.
- [ ] Each program should also have the appropriate background image as shown in the provided project design
- project design
sp
- your design
yd

File: about.html [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+                <li><a href="#" class="desktop-navbar-link">Join</a></li>
+                <li><a href="#" class="desktop-navbar-link">Sponsor</a></li>
+                <li><a href="#" class="desktop-navbar-link">News</a></li>
+                <li><a href="#" class="desktop-navbar-link CC-Campaign-link">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="abt-header-section">
+            <div class="abt-theme">
+                <div class="abt-theme-content">
+                    <p class="header-theme abt-header-theme">"You Never Fail Until You Stop Trying"</p>
```
Comment (Efiamotu-1) [inline about.html]: - [ ] Please kindly center the text on the about page for mobile screens, you can see the red arrows showing a guide for  both your design and the template design   
- project design
ct
- your design
cts

File: about.html [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+            <div class="abt-past-pic-edit">
+                <img src="images/about_2011 1.png" alt="global summit picture">
+            </div>
+            <div class="abt-past-pic-edit">
+                <img src="images/about_2013 1.png" alt="global summit picture">
+            </div>
+        </div>
+    </section>
+
+    <section id="Partner" class="partner-container abt-partner-container">
```
Comment (Efiamotu-1) [inline about.html:125]: - [ ] Your partners section on mobile devices doesn't have the appropriate spacing as seen in the provided image below, please kindly adjust your design.
- project design
pt
- your design
pts

File: about.html [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+        <div class="partner-img">
+            <img src="images/mozilla 1.png" alt="mozilla logo">
+            <img src="images/googlekorea 1.png" alt="google logo">
+            <img src="images/naver 1.png" alt="naver logo">
+            <img src="images/daumkakaologo 1.png" alt="dumkakao logo">
+            <img src="images/airbnb 1.png" alt="airbnb logo">
+        </div>
+    </section>
+
+    <footer class="footer">
```
Comment (Efiamotu-1) [inline about.html]: - [ ] Your about footer for desktop device doesn't have the correct background color as can be seen in the project design provided to you 
- project design
ab-ft

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Ayokunnumi1 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+    <main>
+        <section class="main-program-container">
+            <div>
+                <h2>Main Program</h2>
+                <span class="line"></span>
+            </div>
+            <div class="main-program-content">
+                <div class="main-program-short">
+                    <div class="main-program-mini">
+                        <img src="images/lecture-img.png" alt="">
```
Comment (anagudelogu) [inline index.html:78]: - [ ] You did a good job working on the previous reviewer's suggestions! 💯 However, there's still some room to improve the quality of your changes:

| Template |
|---|
| image |

| Your design |
|---|
| image |

If you see, the spacing is not the same as shown in the template, this visual bug got introduced in the latest changes you made. Please fix it! 🤓 🔧

File: about.html [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+                <li><a href="#" class="desktop-navbar-link">News</a></li>
+                <li><a href="#" class="desktop-navbar-link CC-Campaign-link">CC Campaign</a></li>
+            </ul>
+        </nav>
+        <div class="abt-header-section">
+            <div class="abt-theme">
+                <div class="abt-theme-content">
+                    <p class="header-theme">"You Never Fail Until You Stop Trying"</p>
+                    <h1 class="abt-desktop-h1">Content Creator <br> Global Summit 2023</h1>
+                </div>
```
Comment (anagudelogu) [inline about.html:55]: - [ ] As requested by the previous reviewer, please consider centering the content of the header section on your `About` page. Remember that is important to stick as much as possible to the given design:

|Template|Your design|
|---|---|
| image | image |

Please consider fixing the alignment! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @Ayokunnumi1 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

✅ Great use of GitHub flow. 💯
✅ Your project is documented professionally. 📜
✅ Great UI design. 💻
✅ Descriptive PR title, summary and commit messages. 💬

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**